.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -33,29 +33,30 @@ jobs:
       matrix:
         asset:
           - agent
-          - agent-ctl
-          - cloud-hypervisor
-          - cloud-hypervisor-glibc
-          - coco-guest-components
-          - csi-kata-directvolume
-          - firecracker
-          - genpolicy
-          - kata-ctl
-          - kata-manager
-          - kernel
-          - kernel-confidential
-          - kernel-dragonball-experimental
+#          - agent-ctl
+          - busybox
+#          - cloud-hypervisor
+#          - cloud-hypervisor-glibc
+#          - coco-guest-components
+#          - csi-kata-directvolume
+#          - firecracker
+#          - genpolicy
+#          - kata-ctl
+#          - kata-manager
+#          - kernel
+#          - kernel-confidential
+#          - kernel-dragonball-experimental
```

Comment:
Should this be in draft/`do-not-merge` with all the commented out targets, or were they accidentually left in?

Reply:
Added do-not-merge until all dependencies are done. Good point.